pinc/unicode.inc

```diff
@@ -103,8 +107,11 @@ function utf8_character_name($characters)
  * Split a string into chunks on Unicode script boundaries.
  *
  * Inherited characters are included in the chunk with the base character
+ *
+ * @param string $string
```

Comment:
I could have enforced this type (and several others in this file) but thought it would be better to lower the risk of this PR to just add PHPStan type annotation.

Reply:
For this one we should enforce it in the function def.

Reply:
Fair, added it to the function def. As an aside, `UTF8::str_split` accepts `int|string`, which is why I allowed the ambiguity for now.